### PR TITLE
Customizing Screen ShotReport

### PR DESCRIPTION
**Pull request by ShilpaGopal**
Hi,

Captured screen shots are separated as per the Test Cases and Test methods. List of Test Results are captured at the end of every Test Run. Those Test Results are formatted in report.json. 

Kindly review. Let me know.

### CODE REVIEW

**Comment by saikrishna321** [comment]
@ShilpaGopal i don't see the creation of json been called anywere in the code.

**Comment by SrinivasanTarget** [comment]
Yup function call is missing.Rest looks good to me! May be we can add some comments everywhere.

**Comment by ShilpaGopal** [comment]
Yes , I have not called it anywhere. Because I have executed tests in parallel and distributive only on android. Resultsets  are crated based on android screenshot folder alone. Once we test it on iOS and combination of both android and iOS then can we have it as one of the feature in framework.

**Comment by saikrishna321** [comment]
Srini, can u pls test it on iOS ?

**Comment by SrinivasanTarget** [comment]
Yup will do it tomorrow

**Comment by saikrishna321** [comment]
@ShilpaGopal this fails on running tests on iOS, can u please check creation on frames

**Comment by ShilpaGopal** [comment]
@saikrishna321 , Folders are getting created for iOS for the user captured screenshot . But when a Testcase fails , I can see that the way screen shot captured is different. Even that need to be handled now.

**Comment by SrinivasanTarget** [comment]
LGTM now https://gist.github.com/SrinivasanTarget/fa91e80485eb77ddfd38df38d1af8fde

We can merge now.Post merge @saikrishna321 remove the iPhone folder captured below as iOS screenshot folder is enough for us.

screen shot 2016-05-28 at 11 37 02 pm

**Comment by saikrishna321** [comment]
@ShilpaGopal you need to fix the path for the failed screenshot 
https://github.com/ShilpaGopal/AppiumTestDistribution/blob/master/src/main/java/com/appium/manager/AppiumParallelTest.java#L268

Screenshot is captured with DeviceModel and Name but the report is linked to testMethod name which is correct, you need to rename the fileCreation name on failure to the methodName_failed.png

screen shot 2016-05-30 at 8 07 13 am

And i don't get to see the report.json generated under the project.

File: src/main/java/com/appium/manager/AppiumParallelTest.java [hunk 1]
```diff
@@ -627,11 +627,12 @@ public void screenShotAndFrame(String methodName, String device, String classNam
                     if (files1[i].isFile()) { //this line weeds out other directories/folders
                         Path p = Paths.get(files1[i].toString());
                         String fileName = p.getFileName().toString().toLowerCase();
+                        System.out.println("##########@@@@@@@@@########At the Gate########@@@@@@@!!!!!!!!"+fileName.split(".png")[0]+"@@@@@@@@@@@@$$$$$$$$$$$@@@@@@@@@@"+fileName);
```
Comment (saikrishna321) [inline src/main/java/com/appium/manager/AppiumParallelTest.java]: can we remove all these System.out

**Pull request merged**